run.php

```diff
@@ -167,6 +176,10 @@ function usage() {
 	echo "\nPlease specify a source directory/file using --src option.\n\n";
 	usage();
 }
+if (!$options['max-errors']) {
```

Comment:
@tchule I think this is a bug in the new release, as one is now forced to always set the `max-errors` flag.

The check should be something like this:

```
if (isset($options['max-errors']) && $options['max-errors'] === '')
```

Reply:
Yep, you're right. I change that, thx.

Reply:
I have updated the tag 0.14.7 to include the correction.